docs/conf.py

```diff
@@ -0,0 +1,261 @@
+# Configuration file for the Sphinx documentation builder.
+# Mastering Plone documentation build configuration file
+
+
+# -- Path setup --------------------------------------------------------------
+
+from datetime import datetime
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+# import os
+# import sys
+# sys.path.insert(0, os.path.abspath("."))
+
+
+# -- Project information -----------------------------------------------------
+
+project = "Plone RESTAPI JavaScript Client"
```

Comment:
Narrative text should split the acronymns.

```suggestion
project = "Plone REST API JavaScript Client"
```